docs/source/user/hydrodyn/input_files.rst

```diff
@@ -200,6 +213,82 @@ motion to prevent double counting the contributions from first-order
 structural motion already included in the second-order potential-flow wave 
 excitation.
 
+HydroDyn now supports large but slow (well below wave frequencies) 
+transient platform yaw motion with both strip-theory only and hybrid 
+potential-flow models. To enable this capability, the inputs 
+**PtfmYMod**, **PtfmRefY**, **PtfmYCutoff**, and **NExctnHdg** must 
+be set appropriately. Note that HydroDyn still requires the platform 
+roll and pitch angles to be small, i.e., within +/-15 deg.
+
+To conform with the first- and second-order potential-flow theory, 
+which limits the structure to small displacement about a reference 
+mean position, a constant or slowly varying reference platform yaw 
+orientation must be established. 
+
+Setting **PtfmYMod** = 0 lets HydroDyn use a constant reference yaw 
+angle given by **PtfmRefY** in degrees. In this case, the platform 
+yaw rotation during the simulation, as given by the **PRPYaw** 
+output channel, must stay within +/-15 deg of **PtfmRefY** specified 
+by the user. A severe warning will be displayed if this requirement 
+is not met at any point during the simulation, while still allowing 
+the simulation to continue if possible. With a hybrid potential-flow 
+model, the potential-flow wave excitation input file needs to cover 
+a suitable range of wave headings relative to the platform after a 
+yaw offset of **PtfmRefY** is applied.
+
+Alternatively, **PtfmYMod** = 1 lets HydroDyn update the reference 
+yaw position **PtfmRefY** dynamically based on the low-pass-filtered 
+platform yaw rotation, analogous to the modeling of slow-drift motion 
+with **ExctnDisp** = 2 above. In this case, the **PtfmRefY** input 
+allows the user to specify the initial reference yaw position at 
+**t** = 0. The cutoff frequency of the first-order low-pass filter 
+for platform yaw rotation can be set with **PtfmYCutoff** in Hz. 
+Ideally, **PtfmYCutoff** should be placed between the wave frequency 
+region and the characteristic frequency of any slow but large change 
+in platform heading to filter out as much wave-frequency platform 
+motion as possible while minimizing the phase shift in the low-frequency 
+heading change. Throughout the simulation, the instantaneous 
+platform yaw rotation should stay within +/-15 deg of the now 
+time-dependent **PtfmRefY**. A severe warning will be displayed if 
+this requirement is not met at any point during the simulation, while 
+still allowing the simulation to continue if possible. 
+
+With **PtfmYMod** = 1, HydroDyn requires the first- and second-order 
+potential-flow wave excitation input file(s) to cover the full range 
```

Comment:
I thought second-order must be disabled with PtfmYMod =1?

Reply:
We should still support mean- and slow-drift loads through `MnDrift` or `NewmanApp`. This is explained below, but I added a clarification here as well.